### PR TITLE
Fix invisible worm, worms collating issue, and 'worms not retreating correcly ' issue

### PR DESCRIPTION
**Pull request by xxfast**
Fixed following
 - Fix invisible worm, #89 
 - worms collating issue, #94 
 - and 'worms not retreating correctly ' issue #93

### CODE REVIEW

**Comment by xxfast** [comment]
[Bug Fix; #89 Invisible Worm](https://trello.com/c/5xjudQSE/38-bug-fix-89-invisible-worm)

**Comment by xxfast** [comment]
[Bug fix #94; Worms bunching up in the same place](https://trello.com/c/pJIUvGB8/39-bug-fix-94-worms-bunching-up-in-the-same-place)

**Comment by xxfast** [comment]
[Bug fix #93; Worms not retreating correctly](https://trello.com/c/otY3wOzv/40-bug-fix-93-worms-not-retreating-correctly)

File: Assets/Scripts/WormHead.cs [hunk 1]
```diff
@@ -55,28 +56,28 @@ public class WormHead : WormSegment {
...
+            Health health = collision.gameObject.GetComponent<Health>();
+            if (health != null) {
+				health.dealDamage(Health.DamageType.Worm,attackAmount);
+            }
         }
-		if (collision.gameObject.layer == LayerMask.NameToLayer("Astroid")) { //TODO Isuru: remove hardcoded tag
-
-			//If wworm on screen then retreat when hittinng asteroids
-			var screenCenterWorldCoordinates = Camera.main.ViewportToWorldPoint(new Vector3(0.5f, 0.5f, Camera.main.transform.position.z));
+        if (collision.gameObject.layer == LayerMask.NameToLayer("Astroid")) { //TODO Isuru: remove hardcoded tag
```
Comment (krooq) [inline Assets/Scripts/WormHead.cs]: Typo here in Asteroid tag

File: Assets/Scripts/Asteroid.cs [hunk 2]
```diff
@@ -60,7 +60,7 @@ public class Asteroid : DestructableEntity {
     }
 
     protected void OnCollisionEnter2D(Collision2D collision) {
-        if (!IsSpawnProtected()) {
+		if ( true || !IsSpawnProtected()) {
```
Comment (krooq) [inline Assets/Scripts/Asteroid.cs]: I'm not sure how the spawn protection is bugging the worms but I don't think they need it anyway.

Comment (xxfast) [reply Assets/Scripts/Asteroid.cs]: Oops I think I left that in accidentally. Let me remove that

**Pull request merged**